Examples/Picture in picture Integration/README.md

```diff
@@ -0,0 +1,13 @@
+
+![pip_inactive_screen_shot](https://user-images.githubusercontent.com/23476223/29076763-1f8e4d36-7c5f-11e7-81b2-de838d998639.png)
+
+
+![pip_active_screen_shot](https://user-images.githubusercontent.com/23476223/29076575-9684aeae-7c5e-11e7-976d-c2ff4ce5f291.png)
+
+This example demonstrates how to get picture in picture playback of content video.  The example also demonstrates the configuration setup required by an application to be able to use picture in picture. This configuration involves:
```

Comment:
Split this text in 2 bullets:
- Demonstration of Picture-in-Picture functionality built in OneMobileSDK
... screenshots go here...
- Required configuration setup for the client app:
... configuration instructions ...